### PR TITLE
showing tags on keep card during search + reload page after keeping from extension

### PR DESCRIPTION
**Pull request by martinraison**
@2is10 @stkem

### CODE REVIEW

File: packrat/main.js [hunk 1]
```diff
@@ -1532,11 +1532,10 @@ function awaitDeepLink(link, tabId, retrySec) {
   }
 }
 
-var kifiSiteRe = /^https?:\/\/(?:www\.)?kifi\.com/;
 function updateKifiAppTabs() {
   var prefix = webBaseUri();
   for (var url in tabsByUrl) {
-    if (url.lastIndexOf(prefix, 0) === 0 || kifiSiteRe.test(url)) {
+    if (url.lastIndexOf(prefix, 0) === 0 || url.lastIndexOf('https://preview.kifi.com', 0) === 0) {
```
Comment (martinraison) [inline packrat/main.js:1538]: @2is10 I saw you changed that on the `guide` branch, do you mind if I do it on `master` too?

File: packrat/main.js [hunk 2]
```diff
@@ -633,7 +633,7 @@ api.port.on({
         og: data.og,
         isPrivate: data.how === 'private'
       }, function done(keep) {
-        log('[unkeep:done]', keep);
+        log('[keep:done]', keep);
```
Comment (martinraison) [inline packrat/main.js:636]: assuming that was a typo

File: kifi-backend/modules/shoebox/angular-black/src/layout/rightCol/RightColCtrl.js [hunk 3]
```diff
@@ -166,8 +166,15 @@ angular.module('kifi.layout.rightCol', ['kifi.modal'])
...
-      if (event.data && event.data.bookmarkCount > 0) {
-        $rootScope.$emit('showGlobalModal', 'importBookmarks', event.data.bookmarkCount, event);
+      switch (event.data) {
+        case 'import_bookmarks':
+          if (event.data.bookmarkCount > 0) {
+            $rootScope.$emit('showGlobalModal', 'importBookmarks', event.data.bookmarkCount, event);
+          }
+          break;
+        case 'update_keeps':
+          location.reload();
```
Comment (martinraison) [inline kifi-backend/modules/shoebox/angular-black/src/layout/rightCol/RightColCtrl.js:176]: just reloading the whole page for now (we can probably do better in the future given that we're actually passing a message)

File: packrat/main.js (hunk 1 above)
Comment (2is10) [inline packrat/main.js:1538]: fine by me

**Pull request merged**